Solutions/Tomcat/Hunting Queries/TomcatAbnormalRequestSize.yaml

```diff
@@ -0,0 +1,36 @@
+id: d5e1eb24-487a-11ec-81d3-0242ac130003
+name: Tomcat - Abnormal request size
+description: |
+  'Query shows abnormal request size.'
+severity: Low
+requiredDataConnectors:
+  - connectorId: ApacheTomcat
+    dataTypes:
+      - TomcatEvent
+tactics:
+  - Exfiltration
+  - Collection
+relevantTechniques:
+  - T1530
+  - T1537
+query: |
+  let Average_Body_Bytes = TomcatEvent
+  | where TimeGenerated > ago(30d)
+  | summarize Avg_Size = avg(tolong(HttpResponseBodyBytes))

```

Comment:
Same here, really should bring in the data with the correct types

Reply:
Query works in Sentinel w/o casting tolong().
Casting tolong() was added to pass checks. If there is no casting tolong() in files then checks fail with error